api/projects/migrations/0025_add_change_request_project_permissions.py

```diff
@@ -0,0 +1,61 @@
+# Generated by Django 4.2.15 on 2024-09-17 14:07
+
+from django.db import migrations
+
+from permissions.models import PROJECT_PERMISSION_TYPE
+from projects.permissions import (
+    APPROVE_PROJECT_LEVEL_CHANGE_REQUESTS,
+    MANAGE_PROJECT_LEVEL_CHANGE_REQUESTS,
+    PROJECT_PERMISSIONS,
+)
+
+
+def remove_default_project_permissions(apps, schema_model):
+    PermissionModel = apps.get_model(  # pragma: no cover
+        "permissions", "PermissionModel"
+    )
+    PermissionModel.objects.get(  # pragma: no cover
+        key=MANAGE_PROJECT_LEVEL_CHANGE_REQUESTS
+    ).delete()
+    PermissionModel.objects.get(  # pragma: no cover
+        key=APPROVE_PROJECT_LEVEL_CHANGE_REQUESTS
+    ).delete()
+
+
+def insert_default_project_permissions(apps, schema_model):
+    PermissionModel = apps.get_model("permissions", "PermissionModel")
+    manage_description = approve_description = None
+
+    for project_permission in PROJECT_PERMISSIONS:
+        if project_permission[0] == MANAGE_PROJECT_LEVEL_CHANGE_REQUESTS:
+            manage_description = project_permission[1]
+        elif project_permission[0] == APPROVE_PROJECT_LEVEL_CHANGE_REQUESTS:
+            approve_description = project_permission[1]
+
+    assert manage_description
+    assert approve_description
```

Comment:
This code took me a little while to understand. I don't think we've worried about doing this dynamically in the past. I don't mind having the duplication here really. 

Alternatively, if you really want to keep this, maybe let's add a comment. Or, just define the descriptions as constants alongside the permissions themselves.

Reply:
Ok I've duplicated the descriptions directly in the code